styles/globals.css

```diff
@@ -66,6 +66,18 @@ img {
   margin-top: -6rem;
 }
 
+/* Hide scroll in contact-details copy box*/
+.copybox {
+  scrollbar-width: none; /* Firefox */
+  -ms-overflow-style: none; /* IE 10+ */
+}
+
+.copybox::-webkit-scrollbar {
+  width: 0;
+  height: 0;
+  background: transparent; /* Chrome/Safari/Webkit */
+}
```

Comment:
Please wrap this class inside `@layer components` so it will be properly purged on pages that don't use this class.

Also nitpick: `.copybox` -> `.helper-scroll-overflow` - use a more general-purpose name for utility classes.